rules/endpoints.go

```diff
@@ -311,8 +311,7 @@ func (r *DefaultRuleRenderer) endpointIptablesChain(
 	if dropEncap {
 		rules = append(rules, Rule{
 			Match: Match().ProtocolNum(ProtoUDP).
-				DestPorts(uint16(r.Config.VXLANPort)).
-				VXLANVNI(uint32(r.Config.VXLANVNI)),
+				DestPorts(uint16(r.Config.VXLANPort)),
```

Comment:
So does this mean that pods will not be able to send traffic to any host on this UDP port?
Is that a problem? It seems like that might cause a problem occasionally.
This is on my rough understanding of what is going on here based off its usage at line 66 and the comment above that `Chain for traffic _from_ the endpoint.`

Reply:
Yeah, I think that's right.

I think we're OK with this though - pods shouldn't be sending VXLAN traffic anyway. We default to the standard UDP VXLAN port https://github.com/projectcalico/felix/blob/master/config/config_params.go#L172